core/app/models/spree/product_property.rb

```diff
@@ -8,7 +8,7 @@ class ProductProperty < Spree::Base
 
     with_options inverse_of: :product_properties do
       belongs_to :product, touch: true, class_name: 'Spree::Product'
-      belongs_to :property, class_name: 'Spree::Property'
+      belongs_to :property, touch: true, class_name: 'Spree::Property'
```

Comment:
@mad-eel can we add a test for this? thanks :)

Reply:
Done!